communications/rfc.rst

```diff
@@ -16,13 +16,32 @@ On this page:
 - :ref:`rfc-jira`.
 - :ref:`rfc-cases`.
 - :ref:`rfc-discussion`.
-- :ref:`rfc-exceptions`.
 - :ref:`rfc-formalities`.
 
 .. seealso::
 
    :doc:`rfd` has a similar name, but has to do with convening discussions across DM teams.
 
+.. _rfc-cases:
+
+Standard procedures that require an RFC
+=======================================
+
+The following procedures **always** require an RFC:
+
+- :doc:`/stack/adding-a-new-package`.
+- :doc:`/stack/packaging-third-party-eups-dependencies`, or with Conda.
+- :doc:`Updating change-controlled documentation </project-docs/change-controlled-docs>`.
+  An RFC can be used to directly obtain approval for modifying documents which are change-controlled at the DM level (those with “LDM-” handles).
+  An RFC should also be used to to develop subsystem-level consensus before submitting an LCR to modify documents which are change controlled at the project level (for example, those with “LSE-” or “LPM-” handles).
+- :doc:`/stack/deprecating-interfaces`, including changes to programming interfaces, configuration fields, and removal of packages.
+- Making changes to the data products which are or will be provided by the Data Management System.
+  This includes both changes to construction-era data products (such as intermediate stages in the evolution of an on-disk format) and to operations-era data products (which will normally be covered by an update to the baselined documentation describing the product).
```

Comment:
I'm worried that this could be interpreted to include any debugging output, even a dataset that is not meant to be around when a ticket is finally merged, but also things like plots that may or may not be around for a long time.  

Furthermore, it doesn't distinguish between changes to the form of the data products and their content.  Presumably many algorithmic changes modify the content without changing the form.  Even pure computational performance enhancements arguably "make changes to the [meta]data products".

Reply:
I agree that putting constraints on evolution of construction era intermediate products might be a bit overboard. The set of 'data products' can be unbounded in that context.

Reply:
I do like the "form" emphasis of the rephrasing above.  Could that please be incorporated?  (Note: "_that_ are or will be generated")

Reply:
i have put the new suggested version in